package.json

```
@@ -1436,6 +1436,10 @@
                                 "items": {
                                     "type": "string"
                                 }
+                            },
+                            "projectName": {
+                                "type": "string",
+                                "description": "%vscode-docker.tasks.docker-compose.dockerCompose.projectName%"
```

Just a nit-pick since as-written this will work perfectly, but adding `.description` will make it consistent with the other root-level properties (`up`, `down`, `files`, `envFile`)

```suggestion
                                "description": "%vscode-docker.tasks.docker-compose.dockerCompose.projectName.description%"
```